ui/widgets/src/aptoide/java/com/appcoins/wallet/ui/widgets/GamesBundleComposable.kt

```diff
@@ -0,0 +1,196 @@
+    ) {
+      items(items) { item ->
+        CardItem(gameCardData = item)
+      }
+    }
+  }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun CardItem(
+  gameCardData: GameData
+) {
+  val context = LocalContext.current
+  Card(
+    colors = CardDefaults.cardColors(WalletColors.styleguide_blue_secondary) ,
+    elevation = CardDefaults.cardElevation(4.dp),
+    shape = RoundedCornerShape(8.dp),
+    onClick = { openGame(gameCardData.gamePackage, context) },
+    modifier = Modifier
+      .width(300.dp)
+      .height(150.dp)
+  ) {
+
+    Box(
+      modifier = Modifier.fillMaxSize()
+    ) {
+      AsyncImage(
+        model = gameCardData.gameBackground,
+        contentDescription = "Game Icon",
+        modifier = Modifier.fillMaxSize(),
+        contentScale = ContentScale.Crop
+      )
+      Box(
+        modifier = Modifier
+          .align(Alignment.BottomStart)
+          .fillMaxSize()
+          .background(
+            Brush.verticalGradient(
+              0.3F to WalletColors.styleguide_blue_secondary.copy(alpha = 0.0F),
+              0.75F to WalletColors.styleguide_blue_secondary.copy(alpha = 0.95F),
+              1F to WalletColors.styleguide_blue_secondary.copy(alpha = 0.99F)
+            )
+          )
+        )
+      Row(
+        modifier = Modifier
+          .fillMaxWidth()
+          .align(Alignment.BottomCenter)
+          .padding(12.dp),
+//        horizontalArrangement = Arrangement.Start,
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        Card(
+          colors = CardDefaults.cardColors(WalletColors.styleguide_blue_secondary),
+          elevation = CardDefaults.cardElevation(4.dp),
+          shape = RoundedCornerShape(8.dp),
+          modifier = Modifier
+            .size(52.dp),
+        ) {
+          Box(
+            modifier = Modifier.fillMaxSize(),
+          ) {
+            AsyncImage(
+              model = gameCardData.gameIcon,
+              contentDescription = "Game Icon",
+              modifier = Modifier
+                .size(52.dp),
+              contentScale = ContentScale.Crop
+            )
+          }
+        }
+        Text(
+          text = gameCardData.title,
+          fontSize = 16.sp,
+          fontWeight = FontWeight.Bold,
+          color = WalletColors.styleguide_light_grey,
+          lineHeight = 24.sp,
+          modifier = Modifier
+            .align(Alignment.Bottom)
+            .padding(bottom = 6.dp, start = 20.dp, end = 20.dp)
+        )
+        Spacer(Modifier.weight(1f))
+        Text(
+          text = stringResource(id = R.string.get_button),
+          color = WalletColors.styleguide_pink,
+          fontSize = 16.sp,
+          fontWeight = FontWeight.Bold,
+          modifier = Modifier
+            .align(Alignment.Bottom)
+            .padding(bottom = 6.dp, end = 12.dp)
+        )
+      }
+    }
+  }
+}
+
+private fun openGame(gamePackage: String, context: Context) {
```

Comment:
I'm not confident that theses methods should be in the ui module, is not possible put them on Fragment?

Reply:
It's the same case as in the previous comment, this code can only be included in the Aptoide's Build Variants. So it needs to be inside the GamesBundleComposable.kt file from Aptoide. I could put the entire file in the app module, but seemed better to me to have it here